pylint/plugins/hass_imports.py

```diff
@@ -422,6 +439,11 @@ class HassImportsFormatChecker(BaseChecker):
             "Used when an import from another component should be "
             "from the component root",
         ),
+        "W7425": (
+            "%s should be used via the namespace: %s",
```

Comment:
Hmm, if I were a new contributor and would see that message, I'm not sure I'd know what I should do now. We should improve the message maybe even with the exact import to use instead.

Reply:
```
************* Module homeassistant.components.acmeda.base
homeassistant/components/acmeda/base.py:9:0: W7425: `async_get` should not be imported directly. Please import `homeassistant.helpers.device_registry` as `dr` and use `dr.async_get` (hass-helper-namespace-import)
homeassistant/components/acmeda/base.py:9:0: W7425: `async_get` should not be imported directly. Please import `homeassistant.helpers.device_registry` as `dr` and use `dr.async_get` (hass-helper-namespace-import)
```